manifests/params.pp

```diff
@@ -17,7 +17,12 @@
   $puppetdb_version          = $puppetdb::globals::version
   $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
-  $manage_pg_repo            = true
+
+  if $::osfamily =~ /RedHat|Debian/ {
```

Comment:
@bastelfreak I'll leave that for a Puppet 4 or Facts Hash PR. Wanted to keep the scope of this PR as close as possible to just making the module compatible with `puppetlabs-postgresql` 5.x for now.